source/production/F0.Minesweeper.Logic/Minefield.cs

```diff
@@ -0,0 +1,26 @@
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic
+{
+	public class Minefield : IMinefield
+	{
+		private readonly uint width, height, mineCount;
+
+		internal Minefield(uint width, uint height, uint mineCount)
```

Comment:
I'd prefer to have the `MinefieldOptions` as the input argument of the constructor since it's convenient and easier to extend. What do you think?

Reply:
Sounds good actually. I thought about the ```Minefield``` not needing to know about the ```MinefieldOptions``` but it fits actually, so I'll make a second constructor for that.
My reasoning was that the ```MinefieldFactory``` might set some other properties of the ```Minefield``` in the future according to the ```MinefieldOptions```